object_storage_buckets.go

```diff
@@ -42,8 +42,22 @@ func (i *ObjectStorageBucket) UnmarshalJSON(b []byte) error {
 type ObjectStorageBucketCreateOptions struct {
 	Cluster string `json:"cluster"`
 	Label   string `json:"label"`
+
+	ACL         ObjectStorageACL `json:"acl,omitempty"`
+	CorsEnabled *bool            `json:"cors_enabled,omitempty"`
```

Comment:
pointer bools are always awkward, but I guess necessary here

Reply:
Yeah, it seems like this is the best option unless we abstracted bools in some way.